### ISSUE TITLE
Introduce `padding_idx` to `nn.Embedding`

### ISSUE THREAD

**Issue by mikkelfo** [FGA-DIKU/EHR#166]
Set the `padding_idx` in the `nn.Embedding` inside the `EhrEmbeddings` class which makes the padding_token a zero-vector. Might be redundant, but [ModernBERT does it](https://github.com/huggingface/transformers/blob/752ef3fd4e70869626ec70657a770a85c0ad9219/src/transformers/models/modernbert/modeling_modernbert.py#L202)

From https://pytorch.org/docs/stable/generated/torch.nn.Embedding.html: "If specified, the entries at `padding_idx` do not contribute to the gradient; therefore, the embedding vector at `padding_idx` is not updated during training, i.e. it remains as a fixed “pad”. For a newly constructed Embedding, the embedding vector at `padding_idx` will default to all zeros, but can be updated to another value to be used as the padding vector."
